pytasty/request_handler.py

```diff
@@ -46,7 +49,7 @@ def clean_url(url):
     cleaned_url = splits[0]
     if splits[1:]:
         params = {key: list(set(val))
-                  for key, val in urlparse.parse_qs(splits[1]).items()}
+                  for key, val in list(urllib.parse.parse_qs(splits[1]).items())}
```

Comment:
@Khusbu Can you explain the urlparse issue that could happen for this line?

Reply:
```

Reply:
The snippet you shared should work fine.

Reply:
Cool, thanks 👍